cmd/lmes_driver/main.go

```diff
@@ -54,13 +54,19 @@ var (
 	customCards         strArrayArg
 	customTemplates     strArrayArg
 	customSystemPrompts strArrayArg
+	taskNames           strArrayArg
 	copy                = flag.String("copy", "", "copy this binary to specified destination path")
 	getStatus           = flag.Bool("get-status", false, "Get current status")
 	shutdown            = flag.Bool("shutdown", false, "Shutdown the driver")
 	outputPath          = flag.String("output-path", OutputPath, "output path")
 	detectDevice        = flag.Bool("detect-device", false, "detect available device(s), CUDA or CPU")
 	commPort            = flag.Int("listen-port", driver.DefaultPort, "driver serves APIs on the port")
 	downloadAssetsS3    = flag.Bool("download-assets-s3", false, "Download assets from S3")
+	customTaskGitURL    = flag.String("custom-task-git-url", "", "Git repository URL for custom tasks")
```

Comment:
I am thinking of merging several args into one, i.e., `externalSource`, and its value could be a JSON object that we can marshal/unmarshal using a struct type directly or a format you see fit. To generalize it, one extra info is needed: where these external sources are copied to. And it could accept multiple values, which means you can specify this argument multiple times. With this new argument, the driver can support multiple external resource retrievals.